setup.py

```diff
@@ -35,7 +35,7 @@
         'datarobot_provider.sensors',
         'datarobot_provider.operators',
     ],
-    install_requires=['apache-airflow>=2.3.0', 'datarobot>=3.3.0'],
+    install_requires=['apache-airflow>=2.3.0', 'datarobot>=3.3.1'],
```

Comment:
Do you really need the new version? There were no code changes so why force users to use newer version of library?

Reply:
Bumped minimal datarobot version to 3.3.1 because it contains bugfixes

Reply:
You should try and keep as low a version as possible, it makes it easier for this package to be installed. Unless if this library requires those bug fixes.